### PR TITLE
enabling toggle notification by default

### PR DESCRIPTION
**Pull request by harriott**
1. `logger.vim` had a minor variable error
2. `g:table_mode_verbose`  wasn't declared in the doc

I've fixed these, and then set verbose to be on by default, which I believe would be generally preferred.

### CODE REVIEW

File: plugin/table-mode.vim [hunk 1]
```diff
@@ -16,7 +16,7 @@ endfunction
 
 " Set Global Defaults {{{1
 call s:SetGlobalOptDefault('table_mode_corner', '+')
-call s:SetGlobalOptDefault('table_mode_verbose', 0)
+call s:SetGlobalOptDefault('table_mode_verbose', 1)
```
Comment (dhruvasagar) [inline plugin/table-mode.vim]: I believe `0` should still be the default value for this. Why do you think it should be enabled by default?

File: doc/tags [hunk 2]
```diff
@@ -0,0 +1,61 @@
+:TableAddFormula	table-mode.txt	/*:TableAddFormula*
```
Comment (dhruvasagar) [inline doc/tags]: Please don't add this file, I use ctags too, but I don't think this needs / should be within the repository. I have my tags files ignored globally so I don't run into situations like this and end up accidentally committing it.

File: plugin/table-mode.vim (hunk 1 above)
Comment (harriott) [inline plugin/table-mode.vim]: When starting a new table with no delimiters as yet entered there is no syntax-hilighting indication at all that we've actually entered Table Mode, which gives the impression that it's not functioning. Better to have the verbose message appear as confirmation that `TableModeToggle` has worked.

File: doc/tags (hunk 2 above)
Comment (harriott) [inline doc/tags]: Okay, I too've fixed that, thanks.

File: doc/table-mode.txt [hunk 3]
```diff
@@ -173,6 +174,10 @@ g:loaded_table_mode					*table-mode-loaded*
 	Use this option to disable the plugin: >
 		let g:loaded_table_mode = 1
 <
+g:loaded_table_verbose					*table-mode-verbose*
```
Comment (dhruvasagar) [inline doc/table-mode.txt]: This needs to be `g:table_mode_verbose`

**Comment by dhruvasagar** [comment]
Thanks for your time and patience :)

**Comment by harriott** [comment]
Thanks for your help, that was my first pull request.
Now it's done, any idea how long I should keep my fork on GitHub?

**Comment by dhruvasagar** [comment]
That's up to you. If you're interested you can keep it up and continue making contributions like these.

**Pull request merged**